test/e2e/e2e_suite_test.go

```diff
@@ -6,8 +6,9 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/operator-framework/rukpak/api/v1alpha1"
```

Comment:
Note: I'm fairly surprised the linting check didn't complain about this as we have a goimportas configuration that enforces the rukpakv1alpha1 package import.